apps/desktop/src/shared/context/contributors/internalTask.ts

```
@@ -0,0 +1,42 @@
+}
+
+export const internalTaskContributor: ContextContributor<{
+	kind: "internal-task";
+	id: string;
+}> = {
+	kind: "internal-task",
+	displayName: "Task",
+	description: "Internal task spec inlined as context.",
+	requiresQuery: true,
+	async resolve(source, ctx) {
+		let task;
```

Biome (2.4.11)

[error] This variable implicitly has the any type.

(lint/suspicious/noImplicitAnyLet)